packages/backend/package.json

```diff
@@ -38,6 +38,7 @@
     "watch": "vite --mode development build -w"
   },
   "dependencies": {
+    "@podman-desktop/api": "^0.0.202402061753-5c5d589",
```

Comment:
will the feature in `0.0.202402061753-5c5d589` be in 1.8 ?

Reply:
@benoitf These 2 PRs will be needed by AI Studio in the next release:
- https://github.com/containers/podman-desktop/pull/5862
- https://github.com/containers/podman-desktop/pull/5864

Do I need to 'register' them somewhere, or will they be part of the next release automatically?

Reply:
@benoitf

Reply:
automatic as it's in the main branch and 1.8 will use the main branch

Reply:
but it means you can't cut new releases of AI studio using that API until 1.8 is out or you need to add some 'fallback' in the code

like if (api.engine.listPods) {...} else {....}

Reply:
you should update the entry in devDependencies or it will be duplicated